src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/appointments/AppointmentsResource.kt

```diff
@@ -326,20 +326,20 @@ class AppointmentsResource(private val appointmentService: AppointmentService) {
       ),
       ApiResponse(
         responseCode = "403",
-        description = "Forbidden to access this endpoint when role NOMIS_INCENTIVES not present",
+        description = "Forbidden to access this endpoint when role not present",
         content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
       ),
     ],
   )
   fun getAppointmentsByFilter(
     @PageableDefault(sort = ["eventId"], direction = Sort.Direction.ASC)
     pageRequest: Pageable,
-    @RequestParam(value = "prisonIds", required = false)
+    @RequestParam(value = "prisonIds", required = true)
```

Comment:
Simpler to make prison required than find a way to omit it in the query. Migrations will always be done by prison anyway i think